### ISSUE TITLE
isssue 

### ISSUE THREAD

**Issue by nisha331** [nisha331/flow_cycle1#93]
![page 1](https://user-images.githubusercontent.com/76838660/223807284-88793496-0379-4fef-a15c-573fe6ca3ef9.jpg)

mark the estimated period dates in the calendar .. 
user can select any date , any year and see  probable period dates.
(*here its just done for jan but u need to do for all the months . )

**Comment by Neha00011** [comment]
I want to work on this issue under JWOC.

**Comment by vk20july** [comment]
Mam can you repeat if possible 
unable to understand
Though I want to work on this issue under jwoc just in advance

**Comment by nisha331** [comment]
There is a calendar in home page when u click on that .. a page with calendar appears with all the months and dates . u can move too next year as well . then the input given by the user in the home page is stored , then calculate the period dates and mark it red  for every month (5 days is taken as general).

**Comment by vk20july** [comment]
> ![page 1](https://user-images.githubusercontent.com/76838660/223807284-88793496-0379-4fef-a15c-573fe6ca3ef9.jpg)
> 
> mark the estimated period dates in the calendar .. user can select any date , any year and see probable period dates. (*here its just done for jan but u need to do for all the months . )

so here the user select the 4 jan date then we have to mark the 5 days including this a red  color right?

**Comment by vk20july** [comment]
@nisha331 and does the user want to see that mark dates immediately when they click any date?

**Comment by vk20july** [comment]
@nisha331 mam only clear this doubt because i think i can do this task

**Comment by nisha331** [comment]
> @nisha331 and does the user want to see that mark dates immediately when they click any date?

yes when they enter the date on the home page . dont redirect tehm to this page .. just store the value . when they click on calender then show it
